src/main/java/foo/Foo.java

```diff
@@ -3,14 +3,20 @@
 import bar.Simplest;
 
 /**
- * Foo class
+ * Foo class :)
  */
 public class Foo {
 
+    private static int m;
+    
     public static int times(int a, int b) {
     	// Use Simplest to introduce a dependency
     	int i = Simplest.add(5, 6);
-        return a * b;  
+        
+        // pouet! :-))
+        float f;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "f" local variable. ([squid:S1481](https://rebloch.sonarsource.com/organizations/fab/rules#rule_key=squid%3AS1481))

[See it in SonarCloud](https://rebloch.sonarsource.com/project/issues?id=test%3Afake-project-for-tests&pullRequest=1&issues=AWHcBRtLQYL-lH1H_cJA&open=AWHcBRtLQYL-lH1H_cJA)

[ISSUE_KEY]: AWHcBRtLQYL-lH1H_cJA